src/sdkVersionID/getAnalyticsOptions.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @description Gets the analyticsOptions from options- should include sdkSemver, techVersion, sdkCode, and feature
+ * @param options
+ * @returns {{sdkSemver: (string), sdkCode, feature: string, techVersion: (string)}}
+ */
+export default function getAnalyticsOptions(options) {
+  let analyticsOptions = {
+    sdkSemver: options.sdkSemver,
+    techVersion: options.techVersion,
+    sdkCode: options.sdkCode,
+    feature: '0'
```

Comment:
Suggest to have feature.accessibility: true, etc. so we will have the future option to log several features